packages/okta-react/package.json

```diff
@@ -61,9 +64,18 @@
     "jest": "^22.4.2",
     "polished": "^1.7.0",
     "protractor": "^5.1.2",
-    "react-dom": "^15.6.1",
+    "react": ">=16",
+    "react-dom": ">=16",
     "react-router-dom": "^4.2.2",
     "rimraf": "^2.6.2",
     "styled-components": "^2.1.2"
+  },
+  "jest": {
+    "setupFiles": [
+      "./test/test-setup.js"
```

Comment:
include the test-setup file in all tests for easier testing. Also sets global enzyme testing functions.